docs/samples/browser-plugin-meetings/index.html

```diff
@@ -342,6 +342,7 @@ <h2 class="collapsible">
                       <option value="">Person ID</option>
                       <option value="">SIP URI</option>
                       <option value="CONVERSATION_URL">Conversation URL</option>
+                      <option value="">One to One Meeting</option>
```

Comment:
Why do we have this value? I do not see it being used anywhere in the code

Reply:
@mkesavan13 I had used another approach to resolve the issue by the input of Shreyas instead of keeping condition of Three inputs like Person ID or Email or One to One Meeting (these inputs were ultimately follow same process to create locus call) and SIP URI only option required password status based on the SIP URI i changed the condition.If it is SIP URI and password was not defined the join buttons will disabled and if the option we select other than SIP URI (Email or Person ID or One to One Meeting ) and password is Unknown then the join buttons only enabled.For Password Status is NOT REQUIRED(for Conversational URL) the join button only enabled but we dont use Conversational URL as it follows Instant AdHoc Meeting.

Reply:
Email or Person ID or One to One Meeting follows the Locus Call and Conversation url follow Instant AdHoc Meeting Instant.

Reply:
In which case, we do not need all 3 options there right? Should we consider removing them and just have SIP URI and None?

CC - @Shreyas281299

Reply:
@mkesavan13 ,I think we can have SIP URI and One to One Meeting or None.Instead of Keeping None Can We keep Email or Other Name for the Tag

Reply:
And Also Based on the Option Selected the variable created and based on the variable the condition was working.By keeping only SIP URI by default the option was selected was SIP URI and variable also consider as SIP URI.My Suggestion is the two Keep Drop-down of two SIP URI and second one may be other option in tha above Three

Reply:
But these options are empty and is not even being used anywhere. It's just confusing to have it

Reply:
@mkesavan13 , Yes From my side as Person ID and Email and One to One Meeting was following the locus call approach.In drop-down options we can keep only three options 1)SIP URI 2)Email 3) Person ID .As we are not supported Instance Ad-hoc meeting do we need Conversation URL and For The One-One meeting option and Email we giving same destination input and follow Locus Call Approach.